core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -395,6 +395,15 @@ protected function doReplace($strBuffer, $blnCache)
 							$strTitle = 'Go back';
 						}
 
+						$strBackScript = "<script>document.documentElement.addEventListener('click',function(e){"
+							. "e&&e.target&&e.target.matches('a[href$=\"#_back\"],a[href$=\"#_back\"] *')&&(history.go(-1),e.preventDefault&&e.preventDefault())"
+						. "},true)</script>";
+
+						if (!\in_array($strBackScript, $GLOBALS['TL_HEAD'] ?? array(), true))
```

Comment:
Can't we just use a key here? Like `$GLOBALS['TL_HEAD']['contao_back_insert_tag'] = $strBackScript;`?